fdbclient/NativeAPI.actor.cpp

```diff
@@ -382,6 +382,82 @@ void traceTSSErrors(const char* name, UID tssId, const std::unordered_map<int, u
 	}
 }
 
+/*
+    For each request type, this will produce
+    <Type>Count
+    <Type>{SS,TSS}{Mean,P50,P90,P99}
+    Example:
+    GetValueLatencySSMean
+*/
+void traceSSOrTSSPercentiles(TraceEvent& ev, const std::string name, ContinuousSample<double>& sample) {
+	ev.detail(name + "Mean", sample.mean());
+	ev.detail(name + "P50", sample.median());
+	// don't log the larger percentiles unless we actually have enough samples to log the accurate percentile instead of
+	// the largest sample in this window
+	if (sample.getPopulationSize() > 10) {
```

Comment:
should these be >=?

Reply:
With the old implementation I had, this was necessary to avoid the top value. With:
`int idx = std::max<int>(std::ceil(samples.size() * percentile) - 1, 0)`
as the implementation, >= should be correct